src/main/java/com/atlauncher/mclauncher/LegacyMCLauncher.java

```diff
@@ -407,12 +407,6 @@ public static void main(String[] args) {
                 } catch (InstantiationException e) {
                     System.out.println("Applet wrapper failed! Falling back " + "to compatibility mode.");
                     mc.getMethod("main", String[].class).invoke(null, (Object) mcArgs);
-                } finally {
```

Comment:
`URLClassLoader`'s `close` method was added in Java 7, so this causes `NoSuchMethodError`s on Java 6.